src/ai/backend/manager/api/session.py

```diff
@@ -494,37 +495,40 @@ async def _create(request: web.Request, params: Any) -> web.Response:
             params['bootstrap_script'] = script
 
     try:
-        kernel_id = await asyncio.shield(root_ctx.registry.enqueue_session(
-            session_creation_id,
-            params['session_name'], owner_access_key,
-            [{
-                'image_ref': requested_image_ref,
-                'cluster_role': DEFAULT_ROLE,
-                'cluster_idx': 1,
-                'cluster_hostname': f"{DEFAULT_ROLE}1",
-                'creation_config': params['config'],
-                'bootstrap_script': params['bootstrap_script'],
-                'startup_command': params['startup_command'],
-            }],
-            params['config']['scaling_group'],
-            params['session_type'],
-            resource_policy,
-            domain_name=params['domain'],  # type: ignore  # params always have it
-            group_id=group_id,
-            user_uuid=owner_uuid,
-            user_role=request['user']['role'],
-            cluster_mode=params['cluster_mode'],
-            cluster_size=params['cluster_size'],
-            startup_command=params['startup_command'],
-            session_tag=params['tag'],
-            starts_at=starts_at,
-            agent_list=params['config']['agent_list'],
-        ))
-        resp['sessionId'] = str(kernel_id)  # changed since API v5
-        resp['sessionName'] = str(params['session_name'])
-        resp['status'] = 'PENDING'
-        resp['servicePorts'] = []
-        resp['created'] = True
+        if session_ptask_group is not None:
```

Comment:
Please remove bogus checks like this line while keeping mypy happy.
How could you do it?